modules/service_container_doctrine/lib/Drupal/service_container_doctrine/Tests/ServiceContainerDoctrineIntegrationTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\service_container_doctrine\Tests\ServiceContainerDoctrineIntegrationTest.
+ */
+
+namespace Drupal\service_container_doctrine\Tests;
+
+use Drupal\Component\Plugin\PluginBase;
+use Drupal\service_container\Tests\ServiceContainerIntegrationTestBase;
+use Mockery\CountValidator\Exception;
+use Symfony\Component\Yaml\Exception\RuntimeException;
+
+class ServiceContainerDoctrineIntegrationTest extends ServiceContainerIntegrationTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp('service_container_doctrine_test');
+
+    $this->container = \Drupal::getContainer();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getInfo() {
+    return array(
+      'name' => 'ServiceContainerDoctrineIntegrationTest',
+      'description' => 'Tests the \ServiceContainer class (Doctrine integration)',
+      'group' => 'service_container',
+    );
+  }
+
+  /**
+   * Tests if plugins with annotations are available as services.
+   */
+  public function testDoctrinePlugin() {
+    $plugins = array(
+      array(
+        'module' => 'sc_doctrine_test',
```

Comment:
It is 'owner' not module, also the best practice is to use 'plugin.manager' in Drupal 8.